src/components/ContentActions/index.js

```diff
@@ -56,7 +56,7 @@ function ContentActions({ apiUri, queryParams, type, ...props }) {
     <div className={classes.root}>
       <Section className={classes.section}>
         <Grid container justifyContent="space-between" alignItems="center">
-          <Grid container item alignItems="center">
+          <Grid container item xs={4} alignItems="center">
```

Comment:
```suggestion
          <Grid container item md={4} alignItems="center">
```
1. We _want_ this to take the whole width in small screens,
2. Looks like you're changing this part of the code from 2 different PRs; this and #69 [here](https://github.com/CodeForAfrica/TwoopsTrackerUI/pull/69/files#diff-2071c343700538b0f64649995af238b5baf56a9e79e88e505f5de65c5b6b0d02L58-R84). My advice would be make this change in one of the PRs.